classes/dpkg-bazel.bbclass

```diff
@@ -12,20 +12,30 @@
 inherit dpkg
 
 DEPENDS += "bazel-bootstrap"
+BAZEL_FETCH_DEPENDS += "bazel-bootstrap bazel-bootstrap-data git symlinks"
+
+# ccache + bazel is broken
+USE_CCACHE="0"
 
 # fetch all bazel deps and add to package sources
 bazel_fetch() {
-    dpkg_do_mounts
     E="${@ isar_export_proxies(d)}"
-    sudo chroot --userspec=$( id -u ):$( id -g ) ${BUILDCHROOT_DIR} /bin/bash -c 'cd ${PP}/${PPS} && rm -rf local && ./debian/rules local'
-    dpkg_undo_mounts
+    echo "SCHROOT: ${SBUILD_CHROOT}"
+    schroot -d / -c ${SBUILD_CHROOT} -u root -- sh <<EOF
+        echo ${ISAR_APT_REPO} > /etc/apt/sources.list.d/isar_apt.list;
```

Comment:
do we need to start the inline script with `set -e` to abort on the first error?

Reply:
Yes. good catch.